### ISSUE TITLE
APEX jobs never finish 'Processing'

### ISSUE THREAD

**Issue by smaass4** [douglascayers/sfdc-convert-attachments-to-chatter-files#52]
The issue is that the Batch Apex job always has the status of Processing and never completes.

Background:
* In a sandbox, last week I did the installation and successfully completed several "#2 Convert Attachments".
* Then, over the weekend, Summer Release 18 happened.
* And the sandbox was refreshed.
* I reviewed the prerequisites for "Enable Create Audit Fields" and "Email-to_Case". Both are confirmed to be enabled.
* I deployed the package using 'Install for Admins Only'.
* I assigned myself the "Convert Attachments to Files" permission set
* I have ran one single record test that showed still showing "Processing" status after one hour.  

* Since this didn't seem right, I uninstalled and deleted the package.
* Reinstalled the package with 'Install for All user'.
* Assigned myself the "Convert Attachments to Files" permission set and then switched to the "Convert Attachments to Files" app
* I have run three single record tests but they  are still showing "Processing" status. The first one has been in the job queue for an hour.
The three single records were:
0061300001E1HbMAAV
0061300001C0kRc
a1813000008mJcpAAE (This is a custom object)

What can I do to get the jobs to get to Completed Status?

Apex Jobs looks like this:
![image](https://user-images.githubusercontent.com/3815078/41302931-698041a0-6e31-11e8-80a7-c1dd9d5e7b8d.png)

**Comment by douglascayers** [comment]
It doesn’t look like the job has failed so perhaps it’s waiting on system resources. My thoughts are to open a case with support to see if they can identify what the jobs are hung up on behind the scenes.

**Comment by douglascayers** [comment]
Also, thank you for your interest in my app to convert attachments AND for the very thorough details on what you’ve tried so far :)

**Comment by smaass4** [comment]
It turns out that there is currently a known issue with Partial Sandboxes when viewing Attachments or Content.

Here is the link to track it:
Error 192261835 when viewing Attachments or Content in a Partial Sandbox
https://success.salesforce.com/issues_view?id=a1p3A000000AT9DQAW

**Comment by douglascayers** [comment]
Thanks for the update @smaass4. Curious, did Salesforce support point you to that known issue regarding the batch jobs not completing in your sandbox?

**Comment by smaass4** [comment]
@douglascayers , yes, I had a Salesforce Support case which was quite thorough and helped determine this was the issue.  This was only for my Partial Sandbox.  Our Full Sandbox and Production were fine.
